src/main/java/org/example/Main.java

```diff
@@ -1,22 +1,31 @@
 package org.example;
 import java.util.Scanner;
 
-import static java.lang.System.in;
-import static java.lang.System.out;
+import static java.lang.System.*;
 
 public class Main {
 
     public static final String CORRECT_NUMBER = "5";
+    public static  int MAX_TRIES = 5;
+    public static int NUMBER_TRIES= 0;
```

Comment:
this is not a constant variable - only public static final variables should be written LIKE_THIS
This should be local variable inside the main method